tests/lib/rules/object-curly-newline.js

```diff
@@ -432,6 +432,134 @@ ruleTester.run("object-curly-newline", rule, {
             ].join("\n"),
             options: [{ ObjectExpression: "always", ObjectPattern: "never" }],
             parserOptions: { ecmaVersion: 6 }
+        },
+
+        // "ImportDeclaration" ---------------------------------------------
+        {
+            code: [
+                "import {",
+                "    a,",
+                " b",
+                "} from 'module';"
+            ].join("\n"),
+            options: [{ ImportDeclaration: "always" }],
+            parserOptions: { sourceType: "module" }
```

Comment:
You could add `{ parserOptions: { sourceType: "module" } }` to the RuleTester constructor, in order to avoid having to specify it in each of your tests. I don't think there's a need for any test to specify `{ sourceType: "script" }`, but let me know if I'm mistaken.

Reply:
Thank you for the information. I'm not too sure about `{ sourceType: "script" }`. To be honest, I'm learning from the other tests in the repo and I'm still unsure where to find more information about this.